### PR TITLE
[ja] sync concepts/architecture/control-plane-node-communication.md

### PR DESCRIPTION
**Pull request by KKtheGhost**
## PR Summary

- **EN upstream**: `content/en/docs/concepts/architecture/control-plane-node-communication.md`
- **Sync to**: `content/ja/docs/concepts/architecture/control-plane-node-communication.md`
- **Updates**:
  - Sync'ed upstream `glossary_tooltip`s.
  - Sync'ed upstream `What Next`.

Sync check by`scripts/lsync.sh` on `main` branch:
```diff
$ scripts/lsync.sh content/ja/docs/concepts/architecture/control-plane-node-communication.md
diff --git a/content/en/docs/concepts/architecture/control-plane-node-communication.md b/content/en/docs/concepts/architecture/control-plane-node-communication.md
index 785040cda3..2cfa37d5c5 100644
--- a/content/en/docs/concepts/architecture/control-plane-node-communication.md
+++ b/content/en/docs/concepts/architecture/control-plane-node-communication.md
@@ -11,7 +11,8 @@ aliases:

 <!-- overview -->

-This document catalogs the communication paths between the API server and the Kubernetes cluster.
+This document catalogs the communication paths between the {{< glossary_tooltip term_id="kube-apiserver" text="API server" >}}
+and the Kubernetes {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}}.
 The intent is to allow users to customize their installation to harden the network configuration
 such that the cluster can be run on an untrusted network (or on fully public IPs on a cloud
 provider).
@@ -30,28 +31,28 @@ enabled, especially if [anonymous requests](/docs/reference/access-authn-authz/a
 or [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens)
 are allowed.

-Nodes should be provisioned with the public root certificate for the cluster such that they can
+Nodes should be provisioned with the public root {{< glossary_tooltip text="certificate" term_id="certificate" >}} for the cluster such that they can
 connect securely to the API server along with valid client credentials. A good approach is that the
 client credentials provided to the kubelet are in the form of a client certificate. See
 [kubelet TLS bootstrapping](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/)
 for automated provisioning of kubelet client certificates.

-Pods that wish to connect to the API server can do so securely by leveraging a service account so
+{{< glossary_tooltip text="Pods" term_id="pod" >}} that wish to connect to the API server can do so securely by leveraging a service account so
 that Kubernetes will automatically inject the public root certificate and a valid bearer token
 into the pod when it is instantiated.
 The `kubernetes` service (in `default` namespace) is configured with a virtual IP address that is
-redirected (via `kube-proxy`) to the HTTPS endpoint on the API server.
+redirected (via `{{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}`) to the HTTPS endpoint on the API server.

 The control plane components also communicate with the API server over the secure port.

-As a result, the default operating mode for connections from the nodes and pods running on the
+As a result, the default operating mode for connections from the nodes and pod running on the
 nodes to the control plane is secured by default and can run over untrusted and/or public
 networks.

 ## Control plane to node

 There are two primary communication paths from the control plane (the API server) to the nodes.
-The first is from the API server to the kubelet process which runs on each node in the cluster.
+The first is from the API server to the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} process which runs on each node in the cluster.
 The second is from the API server to any node, pod, or service through the API server's _proxy_
 functionality.

@@ -89,7 +90,7 @@ connections **are not currently safe** to run over untrusted or public networks.

 ### SSH tunnels

-Kubernetes supports SSH tunnels to protect the control plane to nodes communication paths. In this
+Kubernetes supports [SSH tunnels](https://www.ssh.com/academy/ssh/tunneling) to protect the control plane to nodes communication paths. In this
 configuration, the API server initiates an SSH tunnel to each node in the cluster (connecting to
 the SSH server listening on port 22) and passes all traffic destined for a kubelet, node, pod, or
 service through the tunnel.
@@ -117,3 +118,12 @@ connections.
 Follow the [Konnectivity service task](/docs/tasks/extend-kubernetes/setup-konnectivity/) to set
 up the Konnectivity service in your cluster.

+## {{% heading "whatsnext" %}}
+
+* Read about the [Kubernetes control plane components](/docs/concepts/overview/components/#control-plane-components)
+* Learn more about [Hubs and Spoke model](https://book.kubebuilder.io/multiversion-tutorial/conversion-concepts.html#hubs-spokes-and-other-wheel-metaphors)
+* Learn how to [Secure a Cluster](/docs/tasks/administer-cluster/securing-a-cluster/)
+* Learn more about the [Kubernetes API](/docs/concepts/overview/kubernetes-api/)
+* [Set up Konnectivity service](/docs/tasks/extend-kubernetes/setup-konnectivity/)
+* [Use Port Forwarding to Access Applications in a Cluster](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
+* Learn how to [Fetch logs for Pods](/docs/tasks/debug/debug-application/debug-running-pod/#examine-pod-logs), [use kubectl port-forward](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/#forward-a-local-port-to-a-port-on-the-pod)
\ No newline at end of file
```
Sync check by`scripts/lsync.sh` on `sync/ja_control-plane-node-communication` branch:
```diff
$ scripts/lsync.sh content/ja/docs/concepts/architecture/control-plane-node-communication.md
content/ja/docs/concepts/architecture/control-plane-node-communication.md is still in sync
```

Thanks to the reviewers in advance.
Best Regards,

Kivinsae Fang

### CODE REVIEW

**Comment by atoato88** [comment]
The target page on the deploy preview:
- https://deploy-preview-41727--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/architecture/control-plane-node-communication/

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 1]
```diff
@@ -73,3 +73,12 @@ Konnectivityサービスを有効にすると、コントロールプレーン
 
 [Konnectivityサービスのセットアップ](/docs/tasks/extend-kubernetes/setup-konnectivity/)に従って、クラスターにKonnectivityサービスをセットアップしてください。
 
+## {{% heading "whatsnext" %}}
+
+* [Kubernetesコントロールプレーンコンポーネント](/docs/concepts/overview/components/#control-plane-components)について読む。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:78]:
```suggestion
* [Kubernetesコントロールプレーンコンポーネント](/ja/docs/concepts/overview/components/#control-plane-components)について読む。
```
We have already the page translated to Japanese, so use the link to `ja` doc.

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 2]
```diff
@@ -73,3 +73,12 @@ Konnectivityサービスを有効にすると、コントロールプレーン
...
+* [HubsとSpokeモデル](https://book.kubebuilder.io/multiversion-tutorial/conversion-concepts.html#hubs-spokes-and-other-wheel-metaphors)について学習する。
+* [クラスターのセキュリティ](/docs/tasks/administer-cluster/securing-a-cluster/)について学習する。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:80]:
```suggestion
* [クラスターのセキュリティ](/ja/docs/tasks/administer-cluster/securing-a-cluster/)について学習する。
```
ditto.

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 3]
```diff
@@ -73,3 +73,12 @@ Konnectivityサービスを有効にすると、コントロールプレーン
...
+* [Kubernetes API](/docs/concepts/overview/kubernetes-api/)について学習する。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:81]:
```suggestion
* [Kubernetes API](/ja/docs/concepts/overview/kubernetes-api/)について学習する。
```
ditto.

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 4]
```diff
@@ -73,3 +73,12 @@ Konnectivityサービスを有効にすると、コントロールプレーン
...
+* [Konnectivityサービスを設定する](/docs/tasks/extend-kubernetes/setup-konnectivity/)
+* [Port Forwardingを使用してクラスター内のアプリケーションにアクセスする](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
+* [Podログを調べます](/docs/tasks/debug/debug-application/debug-running-pod/#examine-pod-logs)と[kubectl port-forwardを使用します](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/#forward-a-local-port-to-a-port-on-the-pod)について学習する。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:84]:
```suggestion
* [Podログを調べます](/ja/docs/tasks/debug/debug-application/debug-running-pod/#examine-pod-logs)と[kubectl port-forwardを使用します](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/#forward-a-local-port-to-a-port-on-the-pod)について学習する。
```
ditto.

**Review by atoato88 — COMMENTED** [review]
Please check the review comments.

**Comment by KKtheGhost** [comment]
> Please check the review comments.

Hi @atoato88 

Thanks so much for the review.
All review suggestions have been committed and squashed into one commit in `Co-authored-by` mode.

Best Regards,

**Review by nasa9084 — COMMENTED** [review]
/approve
/assign

**Pull request merged**